jmx-metrics/src/integrationTest/resources/hbase/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM dajobe/hbase
+
+# JMX
+EXPOSE 9900
+
+RUN echo "export HBASE_JMX_BASE=\"-Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false\"" >> /opt/hbase/conf/hbase-env.sh
```

Comment:
It looks like we can use copyFileToContainer (putting these files into `resources` instead of using echo commands) instead of a Dockerfile. Like other PR, we want to avoid building docker images as part of running tests

Reply:
Updated

Reply:
Updated